draft-ietf-quic-http.md

```diff
@@ -1478,6 +1478,10 @@ could pose a security risk to an incautious implementer.  An implementation MUST
 ensure that the length of a frame exactly matches the length of the fields it
 contains.
 
+The use of 0-RTT with HTTP/3 creates an exposure to replay attack.  The
```

Comment:
Just a minor suggestion.
Is this really just an HTTP/3 issue? 
The rest of the text refers to "the application protocol" which I believe is more correct.

Reply:
nevermind. I didn't notice this is in the http draft. 
Looks good then.